pkg/signing/signing.go

```diff
@@ -350,9 +368,10 @@ func Verify(cpath, keyServiceURI string, id uint32, isGroup bool, authToken stri
 		}
 		author += fmt.Sprintf("\t%s, Fingerprint %X\n", name, signer.PrimaryKey.Fingerprint)
 	}
+	sylog.Infof("Container is signed")
 	fmt.Printf("Data integrity checked, authentic and signed by:\n%v", author)
 
-	return nil
+	return notLocalKey, nil
```

Comment:
shouldn't this be `return true, nil`?

Reply:
No, this will return `true` if theres no local key matching the signed container. and will return `false` if you have a local key. Line `340` will decided if we return `true`, or `false`.